packages/features/ee/workflows/api/scheduleEmailReminders.ts

```diff
@@ -173,8 +186,10 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
             additionalNotes: reminder.booking.description,
             responses: responses,
             meetingUrl: bookingMetadataSchema.parse(reminder.booking.metadata || {})?.videoCallUrl,
-            cancelLink: `/booking/${reminder.booking.uid}?cancel=true`,
-            rescheduleLink: `/${reminder.booking.user?.username}/${reminder.booking.eventType?.slug}?rescheduleUid=${reminder.booking.uid}`,
+            cancelLink: `${bookerUrl}/booking/${reminder.booking.uid}?cancel=true`,
+            rescheduleLink: `${bookerUrl}/${
+              organizerOrganizationProfile?.username || reminder.booking.user?.username
+            }/${reminder.booking.eventType?.slug}?rescheduleUid=${reminder.booking.uid}`,
```

Comment:
This is the reschedule url used in Confirmation email packages/lib/CalEventParser.ts

` `${calEvent.bookerUrl ?? WEBAPP_URL}/reschedule/${seatUid ? seatUid : Uid}`;`

CancelLink is also different